packages/twenty-front/src/modules/activities/events/components/Events.tsx

```diff
@@ -1,8 +1,29 @@
+import styled from '@emotion/styled';
 import { isNonEmptyArray } from '@sniptt/guards';
 
 import { EventList } from '@/activities/events/components/EventList';
 import { useEvents } from '@/activities/events/hooks/useEvents';
 import { ActivityTargetableObject } from '@/activities/types/ActivityTargetableEntity';
+import AnimatedPlaceholder from '@/ui/layout/animated-placeholder/components/AnimatedPlaceholder';
+import {
+  AnimatedPlaceholderEmptyContainer,
+  AnimatedPlaceholderEmptySubTitle,
+  AnimatedPlaceholderEmptyTextContainer,
+  AnimatedPlaceholderEmptyTitle,
+} from '@/ui/layout/animated-placeholder/components/EmptyPlaceholderStyled';
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+
+const StyledMainContainer = styled.div`
+  align-items: flex-start;
+  align-self: stretch;
+  border-top: ${({ theme }) =>
+    useIsMobile() ? `1px solid ${theme.border.color.medium}` : 'none'};
```

Comment:
I checked the mobile and it's not very convenient because there's no way for the user to read the interesting part.
Could we make this multi-line?

<img width="360" alt="Screenshot 2024-03-24 at 21 49 24" src="https://github.com/twentyhq/twenty/assets/6399865/fa3d4353-d127-4147-af1f-f11e0cbe8954">